packages/contracts-ops/contracts/Config.sol

```diff
@@ -193,25 +240,60 @@ abstract contract Config {
 
     function bridgeTokenUpgrade(string memory domain)
         public
+        override
         onlyInitialized
         returns (Upgrade memory)
     {
         return
             abi.decode(loadBridgeAttribute(domain, "bridgeToken"), (Upgrade));
     }
 
+    function tokenRegistry(string memory domain)
+        public
+        override
+        onlyInitialized
+        returns (TokenRegistry)
+    {
+        return TokenRegistry(address(tokenRegistryUpgrade(domain).proxy));
+    }
+
     function tokenRegistryUpgrade(string memory domain)
         public
+        override
         onlyInitialized
         returns (Upgrade memory)
     {
         return
             abi.decode(loadBridgeAttribute(domain, "tokenRegistry"), (Upgrade));
     }
 
-    function protocol(string memory domain) private returns (string memory) {
-        string memory path = string(abi.encodePacked(".protocol.", domain));
-        return string(vm.parseJson(config, path));
+    function accountant(string memory domain)
+        public
+        override
+        onlyInitialized
+        returns (AllowListNFTRecoveryAccountant)
+    {
+        return
+            AllowListNFTRecoveryAccountant(
+                address(accountantUpgrade(domain).proxy)
+            );
+    }
+
+    function accountantUpgrade(string memory domain)
+        public
+        override
+        onlyInitialized
+        returns (Upgrade memory)
+    {
+        return abi.decode(loadBridgeAttribute(domain, "accountant"), (Upgrade));
+    }
+
+    function ethHelper(string memory domain)
+        public
+        override
+        returns (ETHHelper)
+    {
+        return abi.decode(loadCoreAttribute(domain, "ethHelper"), (ETHHelper));
```

Comment:
this will error if ethHelper does not exist

Reply:
https://github.com/nomad-xyz/monorepo/pull/484/commits/a857e10c6e8bc7d4bc29228593848b37747db6d4